### ISSUE TITLE
Porting twisted.plugins.twisted_trial

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6918]
|<img alt="realcr's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| realcr reported|
|-|-|
|Trac ID|trac#6918|
|Type|enhancement|
|Created|2014-01-18 11:11:17Z|

twisted_trial will support both python2.7 and python3.3 syntax.

Uploaded patch contains:
- Changing implements to implementer regarding zope.interface.

Attachments:

* [plugins_twisted_trial_py3.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/19f/19f0ff5d4dce4afcee3e9c22a732c09d768fcb1e/00e4423bd549a4af755cba41d5bc2f27f740b511.patch) (566 bytes) - added by realcr on 2014-01-18 11:11:47Z - Minor syntax fixes.

<details><summary>Searchable metadata</summary>

```
trac-id__6918 6918
type__enhancement enhancement
reporter__realcr realcr
priority__normal normal
milestone__Python_3_x Python-3.x
branch__ 
branch_author__ 
status__closed closed
resolution__duplicate duplicate
component__core core
keywords__ 
time__1390043477000000 1390043477000000
changetime__1434092717281855 1434092717281855
version__None None
owner__realcr realcr

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun set owner to realcr|
|-|-|

Comments for this are basically the same as for [#6910](https://github.com/twisted/twisted/issues/6910).  Thanks.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_2"></a>|@adiroiban set status to `closed`|
|-|-|

This was fixed in [#7915](https://github.com/twisted/twisted/issues/7915)
